### PR TITLE
Add gopkg make target.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1237/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -115,5 +118,6 @@ help:
 	@echo 'make create-deps - Generate the Go dependencies file.'
 	@echo 'make format - Format the source files.'
 	@echo 'make simplify - Format and simplify the source files.'
+	@echo 'make gopkg - Output the current gpkg repository path and version.'
```
Comment (bac) [inline Makefile]: typo: gopkg

**Comment by bac** [comment]
:+1:

File: Makefile (hunk 1 above)
Comment (frankban) [inline Makefile]: Fixed.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1238/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**